spec/patterns/core_spec.rb

```diff
@@ -1,6 +1,40 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
-require 'logstash/patterns/core'
+require "spec_helper"
+require "logstash/patterns/core"
 
 describe LogStash::Patterns::Core do
+
+  let(:pattern_name) { "SYSLOGLINE"  }
+  let(:value)        { "Mar 16 00:01:25 evita postfix/smtpd[1713]: connect from camomile.cloud9.net[168.100.1.3]" }
+  let(:grok)         { build_grok(pattern_name) }
+
+  it "a pattern pass the grok expression" do
+    expect(grok_match(pattern_name, value)).to pass
+  end
+
+  describe "SYSLOGLINE" do
```

Comment:
How about not having a top level describe of Patterns::Core so we can have multiple "describe pattern" blocks that will not need a subject { } ?
like

```ruby
describe "SYSLOGLINE" do

  let(:value) { "Mar 16 00:01:25 evita postfix/smtpd[1713]: connect from camomile.cloud9.net[168.100.1.3]" }
  it "matches a simple message" do
    expect(subject).to match(value)
  end

  it "generates the program field" do
    expect(grok_match(subject, value)).to include("program" => "postfix/smtpd")
  end
end
```